website/context_processors.py

```diff
@@ -0,0 +1,63 @@
+from django.core.cache import cache
+
+from .utils import get_navigation_click_counts
+
+MENU_ITEMS = {
+    "/companies/": {"name": "Organizations", "icon": "🏛️"},
+    "/sponsor/": {"name": "Sponsor", "icon": "🤝"},
+    "/scoreboard/": {"name": "Scoreboard", "icon": "🏆"},
+    "/leaderboard/": {"name": "Leaderboard", "icon": "🏅"},
+    "/users/": {"name": "Users", "icon": "🧑‍🤝‍🧑"},
+    "/hunts/": {"name": "Bug Bounties", "icon": "🐛"},
+    "/projects/": {"name": "Projects", "icon": "📦"},
+    "/apps/": {"name": "Apps", "icon": "📱"},
+    "/deletions/": {"name": "Deletions", "icon": "🗑️"},
+    "/trademarks/": {"name": "Trademarks", "icon": "™️"},
+    "/bacon/": {"name": "BACON", "icon": "🥓"},
+    "/sitemap/": {"name": "Sitemap", "icon": "🗺️"},
+    "/bltv/": {"name": "BLTV", "icon": "📺"},
+    "/contribute/": {"name": "Contribute", "icon": "✋"},
+    "/contributors/": {"name": "Contributors", "icon": "💻"},
+    "/contributor-stats/": {"name": "Weekly Activity", "icon": "📅"},
+    "/contributor-stats/today/": {"name": "Daily Activity", "icon": "🗓️"},
+    "/about/": {"name": "About Us", "icon": "ℹ️"},
+    "/bidding/": {"name": "Bid on Bugs", "icon": "💰"},
+    "/terms/": {"name": "Terms", "icon": "📜"},
+    "/stats/": {"name": "Stats", "icon": "📊"},
+    "/blt-tomato/": {"name": "BLT Tomato", "icon": "🍅"},
+    "/view-suggestion/": {"name": "Suggestions", "icon": "💬"},
+    "/status/": {"name": "Status", "icon": "✅"},
+    "/": {
+        "name": "Issues",
+        "icon": "⚠️",
+    },
+}
+
+
+def navigation_click_counts(request):
+    cache_key = "navigation_click_counts"
+    click_counts = cache.get(cache_key)
+
+    if not click_counts:
+        click_counts = get_navigation_click_counts()
+        cache.set(cache_key, click_counts, 60 * 15)
+
+    menu_items_dict = {}
+    for item in click_counts:
+        if item["path"] in MENU_ITEMS:
```

Comment:
The problem is that `Issues` is on the root path, so when I log the data for the issue tab it doesn't log anything because it is on the root path `http://127.0.0.1:8000/`.

Reply:
Also, the IP model `class IP(models. Model):` is already in use for another purpose, I think it would be better if we keep a separate model for clicks, it would be a straightforward model with 2/3 fields only.

This will also prevent the original IP model from getting cluttered with click events.